core/src/main/java/rdx/works/core/sargon/CryptoParametersExtensions.kt

```diff
@@ -0,0 +1,43 @@
+package rdx.works.core.sargon
+
+import com.radixdlt.sargon.DerivationPathScheme
+import com.radixdlt.sargon.FactorSourceCryptoParameters
+import com.radixdlt.sargon.Slip10Curve
+import com.radixdlt.sargon.SupportedCurves
+import com.radixdlt.sargon.extensions.contains
+import com.radixdlt.sargon.extensions.init
+
+val FactorSourceCryptoParameters.Companion.babylon: FactorSourceCryptoParameters
+    get() = FactorSourceCryptoParameters(
+        supportedCurves = SupportedCurves.init(Slip10Curve.CURVE25519),
+        supportedDerivationPathSchemes = listOf(DerivationPathScheme.CAP26)
+    )
+
+val FactorSourceCryptoParameters.Companion.olympia: FactorSourceCryptoParameters
+    get() = FactorSourceCryptoParameters(
+        supportedCurves = SupportedCurves.init(Slip10Curve.SECP256K1),
+        supportedDerivationPathSchemes = listOf(DerivationPathScheme.BIP44_OLYMPIA)
+    )
+
+val FactorSourceCryptoParameters.Companion.olympiaBackwardsCompatible: FactorSourceCryptoParameters
+    get() = FactorSourceCryptoParameters(
+        supportedCurves = SupportedCurves.init(Slip10Curve.SECP256K1, Slip10Curve.CURVE25519),
+        supportedDerivationPathSchemes = listOf(DerivationPathScheme.CAP26, DerivationPathScheme.BIP44_OLYMPIA)
+    )
+
+val FactorSourceCryptoParameters.Companion.trustedEntity: FactorSourceCryptoParameters
```

Comment:
is this used? what is purpose of that property?

Reply:
Right I "translated" this in sargon, but since this was used in Android Profile's `OffDeviceMnemonic` this is not used anymore.